test/index.spec.js

```diff
@@ -0,0 +1,265 @@
+const sinon = require('sinon');
+const express = require ('express');
+const proxyquire = require('proxyquire').noPreserveCache();
+const fs = require('fs');
+const bq =  require('@google-cloud/bigquery');
+const http = require('http');
+const testData = require('./testTableMap.json');
+const sandbox = sinon.createSandbox();
+
```

Comment:
I think you were exploring some nice ideas by trying to create "init" functions for everything, but I think readability has suffered. I find this hard to read. 

1. I'm not sure why you would need to proxyquire the same module different times. In general, the module is the 'unit under test', which means you need to stub its dependencies. Once they're stubbed, the stubs should be able to take any action. 

2. In general, I prefer with each test to see, inside the test, what all mocks/stubs will return. Even if it's just a declared variable, it's nice to see that explicitly. With tests, I always recommend more verboseness than with code. They should read like documentation of the code, which is one of their great purposes, and it should be really clear to everyone what IS tested and what IS NOT tested.